templates/router/header.conf.erb

```diff
@@ -24,5 +24,10 @@ router {
     id: <%= scope['qpid::router::router_id'] %>
     mode: <%= scope['qpid::router::router_mode'] %>
     worker-threads: <%= scope['qpid::router::worker_threads'] %>
+    <% unless [nil, :undefined, :undef, ''].include?(scope['qpid::router::hello_interval']) -%>
+    helloInterval: <%= scope['qpid::router::hello_interval'] %>
+    <% end %>
```

Comment:
This should be `<% end -%>` (and below as well). I believe your whitespace issue is then fixed in the tests.